packages/solid-crs-manage/lib/features/loan/loan.states.ts

```diff
@@ -0,0 +1,56 @@
+import { StateSchema } from 'xstate';
+import { LoanContext, WithRequest, WithRequests } from './loan.context';
+
+export enum LoanStates {
+  /**
+   * Initial state: retrieving requests from main inbox
+   */
+  LOADING_LOAN_REQUESTS = '[LoanState: Loading Loan Requests]',
+  /**
+   * The user is viewing all incoming requests
+   */
+  LOAN_REQUEST_OVERVIEW = '[LoanState: Loan Request Overview]',
+  /**
+   * The user is viewing a single incoming request
+   */
+  LOAN_REQUEST_DETAIL = '[LoanState: Loan Request Detail]',
+  /**
+   * The user is creating a new loan request
+   */
+  CREATING_LOAN_REQUEST = '[LoanState: Creating Loan Request]',
```

Comment:
I'll edit this in the spec as well, but it's probably better to rename this state to `LOAN_REQUEST_CREATION` to avoid confusion with `SENDING_LOAN_REQUEST`

(this way the states are divided into more 'idle' and 'active' states, where the 'idle' are meant to display some page and wait for user input, and the 'active' states, starting with a present participle (-ing form), indicate the machine is doing something, and will finish on its own.)